webapp/client/src/components/ContentPageBox.js

```diff
@@ -0,0 +1,147 @@
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import ButtonAnchor from "./ButtonAnchor";
+import SecondaryAnchorButton from "./SecondaryAnchorButton";
+
+const Box = styled.div`
+  background-color: var(--beige-200);
+  padding-top: var(--spacing-09);
+  padding-bottom: var(--spacing-09);
+  margin-top: var(--spacing-11);
+`;
+const InnerBox = styled.div`
+  padding-left: var(--spacing-03);
+  padding-right: var(--spacing-03);
+  margin: 0 auto;
+  max-width: var(--pages-max-width);
+
+  @media (min-width: 768px) {
+    display: grid;
+    grid-template-columns: 1fr 0.5fr 0.5fr 1fr;
+    grid-template-rows: 1fr;
+  }
+
+  @media (min-width: 769px) {
+    margin: 0 auto;
+    padding-left: var(--spacing-06);
+    padding-right: var(--spacing-06);
+  }
+
+  @media (min-width: 1025px) {
+    padding-left: var(--spacing-08);
+    padding-right: var(--spacing-08);
+    grid-gap: 1rem;
+  }
+`;
+
+const TextBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+
+  @media (min-width: 768px) {
+    grid-column: 1 / 4;
+    grid-row: 1;
+  }
+
+  @media (min-width: 1224px) {
+    grid-column: 1 / 3;
+  }
+`;
+
+const ButtonBox = styled.div`
+  display: flex;
+
+  @media (max-width: 576px) {
+    flex-direction: column;
+  }
+`;
+
+const BoxText = styled.span`
+  padding-bottom: var(--spacing-06);
+`;
+
+export default function ContentPageBox({ boxText, anchor, plausibleDomain }) {
```

Comment:
Good questions. I initially thought to incorporate the box direclty in the page but then thought as we did with the infoBox component and the vorbereiten pages where we not only use it across all those subpages but also in the infoForRelatives page passing different props. My assumption was that a page like this will be repeated and potentially have the same or different text fromw within at the bottom of the page. 

Why do we need to be aware of the order? Your right if we create a new page with the same content then it makes sense to move that into the comoponent itself, i wanted to leave room to reuse it with different text/button urls etc and of course the plausible goal names.

Reply:
The InfoBox looks good and it definitely demonstrates the usage of component re-usability. The main difference with the new ContentPageBox component is that the InfoBox has 1 headline, 1 text and 1 anchor. So this is the specification of that reusable component, whenever we need a box with those elements we will be able to reuse it (of course always in the order headline, text and anchor).

But the ContentPageBox has (as defined in the props) 2 texts and 3 anchors meaning every time you re-use that component you need to pass those. And even the anchor definitions and so hard-coded since the component itself expects and anchor called "eligibility", another called "faq" and the last one called "contact". So the only possible anchors you could pass would be exactly those ones (or at least the need to be called that way)

Reply:
Then yes we can reuse that component as you developed, BUT we should remove the naming of the anchors right? because the component is expecting an anchor called "eligibility", another called "faq" and the last one called "contact". So this means that we will always be forced to passed those anchor names

Reply:
Good point 🙂